optaplanner-core/src/test/java/org/optaplanner/core/impl/testdata/domain/scenario/simplequeens/TestdataQueenEntity.java

```diff
@@ -0,0 +1,41 @@
+package org.optaplanner.core.impl.testdata.domain.scenario.simplequeens;
+
+import org.optaplanner.core.api.domain.entity.PlanningEntity;
+import org.optaplanner.core.api.domain.variable.PlanningVariable;
+import org.optaplanner.core.impl.testdata.domain.TestdataEntity;
+import org.optaplanner.core.impl.testdata.domain.scenario.simplequeens.solution.TestdataQueenDifficultyWeightFactory;
+import org.optaplanner.core.impl.testdata.domain.scenario.simplequeens.solution.TestdataRowStrengthWeightFactory;
+
+@PlanningEntity(difficultyWeightFactoryClass = TestdataQueenDifficultyWeightFactory.class)
+public class TestdataQueenEntity extends TestdataEntity {
```

Comment:
I used nqeens because it is very easy to tests CH (it is very simple to see the error if it occurs, also it's very simple write those tests)